metatests.js

```diff
@@ -23,3 +23,19 @@ module.exports = {
   testAsync,
   walk,
 };
+
+if (process.browser) {
+  process.exit = code => {
+    /* eslint-disable no-undef*/
+    __karma__.info({ total: 1 });
+    __karma__.result({ success: !code });
+    __karma__.complete();
+    /* eslint-enable*/
+  };
+  metatests.runner.instance.on('finish', (code) => {
```

Comment:
PTAL https://github.com/metarhia/metatests/pull/75#discussion_r213939103. GitHub has hidden it for some reason.